examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -21072,6 +21072,304 @@
           ]
         }
       ]
+    },
+    {
+      "name": "Anonymous Endpoint Type",
+      "deviceTypeName": null,
+      "deviceTypeCode": null,
```

Comment:
You probably need to actually define a device here...  That's what causes some of the compilation issues.

Reply:
Added a device type with a ID that seems to be reasonable. Thanks.